test/controllers/assumed/create/SubmissionConfirmationControllerSpec.scala

```diff
@@ -17,27 +17,63 @@
 package controllers.assumed.create
 
 import base.SpecBase
+import connectors.{PlatformOperatorConnector, SubscriptionConnector}
+import models.operator.responses.PlatformOperator
+import models.operator.{AddressDetails, ContactDetails}
 import models.submission.AssumedReportSummary
+import models.subscription.{Individual, IndividualContact, SubscriptionInfo}
 import play.api.i18n.Messages
 import play.api.inject.bind
 import play.api.test.FakeRequest
 import play.api.test.Helpers.*
+import org.mockito.ArgumentMatchers.{any, eq as eqTo}
+import org.mockito.Mockito.{never, verify, when}
+import org.mockito.Mockito
+import org.scalatest.BeforeAndAfterEach
+import org.scalatestplus.mockito.MockitoSugar
 import queries.AssumedReportSummaryQuery
 import viewmodels.checkAnswers.assumed.create.AssumedReportCreatedSummary
 import views.html.assumed.create.SubmissionConfirmationView
+import scala.concurrent.Future
 
 import java.time.{Clock, Instant, Year, ZoneId}
 
 
-class SubmissionConfirmationControllerSpec extends SpecBase {
+class SubmissionConfirmationControllerSpec extends SpecBase with MockitoSugar with BeforeAndAfterEach {
 
   private val now = Instant.now()
   private val fixedClock = Clock.fixed(now, ZoneId.systemDefault())
-
+  
+  private val mockPlatformOperatorConnector = mock[PlatformOperatorConnector]
+  private val mockSubscriptionConnector = mock[SubscriptionConnector]
+  
+  override def beforeEach(): Unit = {
+    super.beforeEach()
+    Mockito.reset( mockSubscriptionConnector, mockPlatformOperatorConnector)
+  }
   "SubmissionConfirmation Controller" - {
 
     "onPageLoad" - {
+      val contact = IndividualContact(Individual("first", "last"), "tax1@team.com", None)
+      val subscription = SubscriptionInfo(
+        id = "dprsId",
+        gbUser = true,
+        tradingName = None,
+        primaryContact = contact,
+        secondaryContact = None
+      )
 
+      val operator = PlatformOperator(
+        operatorId = "operatorId",
+        operatorName = "operatorName",
+        tinDetails = Nil,
+        businessName = None,
+        tradingName = None,
+        primaryContactDetails = ContactDetails(None, "name", "tax2@team.com"),
+        secondaryContactDetails = None,
+        addressDetails = AddressDetails("line 1", None, None, None, None, Some("GB")),
+        notifications = Nil
+      )
       "must return OK and the correct view" in {
```

Comment:
Do we need to add tests for when either the getSubscription or viewPlatformOperator returns future failed?